### DISCUSSION TITLE
Wanted. Invitation to Slack

### DISCUSSION THREAD

**Discussion by cfox570** [nasa-jpl/open-source-rover#255] [Q&A] [answered]
How do I join the slack channel?

**Comment by SConaway** [comment, marked as answer]
@cfox570 here's an invite link: https://join.slack.com/t/opensourcerover/shared_invite/zt-mfe0viua-FhcYsNb3S9vUra99uF3IBw

**Reply by apollokit** [reply]
Thanks!
